bin/run.sh

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+
+set -e
+
+unset CDPATH
+cd "$( dirname "${BASH_SOURCE[0]}" )/.."
+
+echo() { builtin echo "[SCITRAN] $@"; }
+
+
+set -o allexport
+
+if [ "$#" -eq 1 ]; then
+    EXISTING_ENV=$(env | grep "SCITRAN_" | cat)
+    source "$1"
+    eval "$EXISTING_ENV"
+fi
+if [ "$#" -gt 1 ]; then
+    echo "Usage: $0 [config file]"
+    exit 1
+fi
+
+
+# Default config values
+if [ -z "$SCITRAN_RUNTIME_HOST" ]; then
+    SCITRAN_RUNTIME_HOST="127.0.0.1"
+fi
+if [ -z "$SCITRAN_RUNTIME_PORT" ]; then
+    SCITRAN_RUNTIME_PORT="8080"
+fi
+if [ -z "$SCITRAN_RUNTIME_PATH" ]; then
+    SCITRAN_RUNTIME_PATH="./runtime"
+fi
+if [ -z "$SCITRAN_RUNTIME_SSL_PEM" ]; then
+    SCITRAN_RUNTIME_SSL_PEM=""
+fi
+if [ -z "$SCITRAN_RUNTIME_BOOTSTRAP" ]; then
+    SCITRAN_RUNTIME_BOOTSTRAP="bootstrap.json"
+fi
+if [ -z "$SCITRAN_PERSISTENT_PATH" ]; then
+    SCITRAN_PERSISTENT_PATH="./persistent"
+fi
+if [ -z "$SCITRAN_PERSISTENT_DATA_PATH" ]; then
+    SCITRAN_PERSISTENT_DATA_PATH="$SCITRAN_PERSISTENT_PATH/data"
+fi
+if [ -z "$SCITRAN_PERSISTENT_DB_PATH" ]; then
+    SCITRAN_PERSISTENT_DB_PATH="$SCITRAN_PERSISTENT_PATH/db"
+fi
+if [ -z "$SCITRAN_PERSISTENT_DB_PORT" ]; then
+    SCITRAN_PERSISTENT_DB_PORT="9001"
+fi
+if [ -z "$SCITRAN_PERSISTENT_DB_URI" ]; then
+    SCITRAN_PERSISTENT_DB_URI="mongodb://localhost:$SCITRAN_PERSISTENT_DB_PORT/scitran"
+fi
```

Comment:
These can be reduced to one-liners:

```bash
SCITRAN_RUNTIME_HOST=${SCITRAN_RUNTIME_HOST:-"127.0.0.1"}
SCITRAN_RUNTIME_PORT=${SCITRAN_RUNTIME_PORT:-"8080"}
SCITRAN_RUNTIME_PATH=${SCITRAN_RUNTIME_PATH:-"./runtime"}
SCITRAN_RUNTIME_SSL_PEM=${SCITRAN_RUNTIME_SSL_PEM:-""}
SCITRAN_RUNTIME_BOOTSTRAP=${SCITRAN_RUNTIME_BOOTSTRAP:-"bootstrap.json"}
SCITRAN_PERSISTENT_PATH=${SCITRAN_PERSISTENT_PATH:-"./persistent"}
SCITRAN_PERSISTENT_DATA_PATH=${SCITRAN_PERSISTENT_DATA_PATH:-"$SCITRAN_PERSISTENT_PATH/data"}
SCITRAN_PERSISTENT_DB_PATH=${SCITRAN_PERSISTENT_DB_PATH:-"$SCITRAN_PERSISTENT_PATH/db"}
SCITRAN_PERSISTENT_DB_PORT=${SCITRAN_PERSISTENT_DB_PORT:-"9001"}
SCITRAN_PERSISTENT_DB_URI=${SCITRAN_PERSISTENT_DB_URI:-"mongodb://localhost:$SCITRAN_PERSISTENT_DB_PORT/scitran"}
```

Question: why both default env vars, **and** a default config map in python?
If that is necessary, add comments to both sections saying that each must be kept in sync with the other.